crates/autopilot/src/run_loop.rs

```diff
@@ -311,8 +310,12 @@ impl RunLoop {
     }
 
     /// Runs the solver competition, making all configured drivers participate.
-    async fn competition(&self, id: AuctionId, auction: &Auction) -> Vec<Participant<'_>> {
-        self.persistence.store_auction(id, auction);
+    async fn competition(
+        &self,
+        id: domain::AuctionId,
+        auction: &domain::Auction,
+    ) -> Vec<Participant<'_>> {
+        self.persistence.archive_auction(id, auction);
```

Comment:
I'd remove this from here (seems like this should happen when we write the auction also into the db)

Reply:
removed